apps/native/app/src/utils/lifecycle/setup-event-handlers.ts

```diff
@@ -126,9 +126,10 @@ export function setupEventHandlers() {
         if (lockScreenComponentId) {
           if (
             lockScreenActivatedAt !== undefined &&
+            lockScreenActivatedAt !== null &&
```

Comment:
~nit: You could use `!isDefined(lockScreenActivatedAt)` utility func instead of those two lines